internal/command/testdata/test/invalid-cleanup-warnings/main.tftest.hcl

```
@@ -0,0 +1,13 @@
+# main.tftest.hcl
+
+run "test" {
+  variables {
+    input = "Hello, world!"
+    validation = "Hello, world!"
+  }
+  assert {
+    condition = test_resource.resource.value == var.validation
```

```suggestion
    condition = test_resource.resource.value == "Hello, world!"
```

We need to make sure we are not referencing the `validation` variable anywhere - this is to make sure we do actually produce the warning during the testing phase.